pkg/store/forward/forward.go

```diff
@@ -109,7 +109,12 @@ func (s *Store) WriteMetrics(ctx context.Context, p *store.PartitionedMetrics) e
 				Observe(time.Since(begin).Seconds())
 
 			if resp.StatusCode/100 != 2 {
-				return fmt.Errorf("response was not 200 OK, but %s", resp.Status)
+				mean := timeseriesMeanDrift(timeseries, time.Now().Unix()*1000)
+				return fmt.Errorf("response status code is %s - mean drift from now for clusters %S is: %.3fms",
```

Comment:
`%S` should be `%s`

Reply:
also, looking at it again, `timeseriesMeanDrift`, returns a value in seconds, not millis, so it should be `%.3fs`.